tests/e2e/pageObjects/browser-page.ts

```diff
@@ -22,6 +22,7 @@ export class BrowserPage {
     //*The following categories are ordered alphabetically (Alerts, Buttons, Checkboxes, etc.).
     //-------------------------------------------------------------------------------------------
     //BUTTONS
+    myRedisDbIcon = Selector('[data-testid = my-redis-db-icon]');
```

Comment:
no need to add extra spaces around '=' here ('[data-testid **=** my-redis-db-icon]')